vehicle/views/vehicle_views.xml

```diff
@@ -0,0 +1,134 @@
+        <field name="name">vehicle.vehicle.tree</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="arch" type="xml">
+            <tree string="Vehicles" decoration-muted="not active">
+                <field name="license_plate"/>
+                <field name="name"/>
+                <field name="user_id"/>
+                <field name="active" invisible="1"/>
+            </tree>
+        </field>
+    </record>
+    <record id="view_vehicle_vehicle_kanban" model="ir.ui.view">
+        <field name="name">vehicle.vehicle.kanban</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="priority" eval="100" />
+        <field name="arch" type="xml">
+            <kanban class="oe_background_grey o_kanban_dashboard">
+                <field name="license_plate" />
+                <field name="name" />
+                <field name="user_id" />
+                <field name="active" />
+                <templates>
+                    <t t-name="kanban-box">
+                        <div class="oe_kanban_global_click">
+                            <div t-attf-class="o_kanban_card_header">
+                                <div class="o_kanban_card_header_title">
+                                    <field name="license_plate"/>
+                                </div>
+                                <div class="container o_kanban_card_content">
+                                    <div class="row">
+                                        <div class="o_kanban_primary_left" style="padding-top:20px;">
+                                            <field name="user_id"/>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </t>
+                </templates>
+            </kanban>
+        </field>
+    </record>
+
+    <record id="view_vehicle_vehicle_form_user" model="ir.ui.view">
+        <field name="name">vehicle.vehicle.form (user_id user)</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="inherit_id" ref="vehicle.view_vehicle_vehicle_form" />
+        <field name="groups_id" eval="[(4, ref('vehicle.group_vehicle_user'))]" />
+        <field name="arch" type="xml">                
+            <field name="user_id" position="attributes">
+                <attribute name="readonly">1</attribute>                   
+            </field>                
+        </field>
+    </record>
+    <record id="view_vehicle_vehicle_form_manager" model="ir.ui.view">
+        <field name="name">vehicle.vehicle.form (user_id manager)</field>
+        <field name="model">vehicle.vehicle</field>
+        <field name="inherit_id" ref="vehicle.view_vehicle_vehicle_form" />
+        <field name="groups_id" eval="[(4, ref('vehicle.group_vehicle_manager'))]" />
+        <field name="priority" eval="25"/>
```

Comment:
Added this lower priority for this view. With this addtion, Manager role (which is User too) will always have editable permissions. With the same priority as User view, there was a random behavior (if user view was applied after, an issue was raised)